modules/ting/ting.admin.inc

```diff
@@ -136,7 +136,7 @@ function ting_admin_reservable_settings($form_state) {
   $form['reservation_settings'] = array(
     '#type' => 'fieldset',
     '#title' => t('Reservation settings'),
-
+    '#tree' => FALSE,
```

Comment:
Is this needed?

Reply:
Last time [you questioned why I removed it](https://github.com/ding2/ding2/pull/1139#discussion_r292157752). Then i told you it was because it's default, but that maybe it would be good to keep for clarity. No response, but no biggie I just added it again.